addons/mass_mailing_sale/views/mailing_mailing_views.xml

```diff
@@ -25,4 +25,29 @@
     		</xpath>
         </field>
     </record>
+
+    <record id="mailing_mailing_invoice_view_search" model="ir.ui.view">
```

Comment:
Same remark, you are adding a new default view. Would propagate lead view comments on this one :)